doc/languages-frameworks/agda.section.md

```diff
@@ -2,34 +2,126 @@
 
 ## How to use Agda
 
-Agda can be installed from `agda`:
+Agda is available as the [agda](https://search.nixos.org/packages?channel=unstable&show=agda&from=0&size=30&sort=relevance&query=agda)
+package and can be installed using `nix-env` although creating a `shell.nix` file in your project is preffered
```

Comment:
It seems strange to write this when this doesn't seem to be the standard use case (I don't think I have ever used nix-shell for installing agda, and in fact doing this requires some more stuff to get it working with an editor?)

Reply:
I'd also say it's best to simply offer different versions on how to do things, and leave the choice to the user

Reply:
@alexarice I'm a bit confused then. If you don't use `nix-env` nor `nix-shell` how do you install Agda? Do you have an override in `$HOME/.config/nixpkgs`?

Reply:
I put it in my `configuration.nix`

Reply:
> It seems strange to write this when this doesn't seem to be the standard use case

What should be considered standard use case then?

a. `nix-shell`,
b. `nix-env` or
c. `$HOME/.config/nixpkgs/configuration.nix`.

IIUC the intention of the first section is just about installing Agda. In which case I would vote for c. Maybe we should start the section with

> This section describes how to use Nix to just install Agda. If you also want to build Agda projects using Nix as well see "Writing Agda packages".
> ... [mention configuration.nix]

Reply:
By (c) I meant `/etc/configuration.nix` for a nixos system. However I'm not really sure that there is a "standard use case", which is part of the reason I'd prefer to just specify the derivation here and not how to install it

Reply:
I agree, there is no standard use case.

Reply:
I just removed how to install the package.